gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -159,6 +157,37 @@ void ROOT::Experimental::TWebWindowsManager::Unregister(ROOT::Experimental::TWeb
    //   }
 }
 
+//////////////////////////////////////////////////////////////////////////
+/// Provide URL address to access specified window from inside or from remote
+
+std::string ROOT::Experimental::TWebWindowsManager::GetUrl(ROOT::Experimental::TWebWindow &win, bool remote)
+{
+   if (!fServer) {
+      R__ERROR_HERE("GetUrl") << "Server instance not exists";
+      return "";
```

Comment:
`return {};` might be better (doesn't need to actually construct a string from a `const char*`).